apps/angular/simple-animations/src/app/paragraph.component.html

```diff
@@ -0,0 +1,4 @@
+<div [@myAnimation]="data">
```

Comment:
Do you need this data reference to get the animation work? maybe just `@myAnimation` works too